src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/TextBufferExtensions.cs

```diff
@@ -32,17 +32,6 @@ public static bool TryGetHostDocumentSyncVersion(this ITextBuffer textBuffer, ou
             return result;
         }
 
-        public static bool IsRazorLSPBuffer(this ITextBuffer textBuffer)
```

Comment:
>IsRazorLSPBuffer [](start = 27, length = 16)

This is Razor-specific and references RazorLSPConstants so I created RazorTextBufferExtensions in Microsoft.VisualStudio.LanguageServerClient.Razor and moved this method there.